# mozbuild/mozbuild/virtualenv.py on windows looks for python.exe but the python script that setup the virtualenv does not add ".exe" to python script
--- mozilla-central/python/virtualenv/virtualenv.py 2017-05-22 10:10:25.999381469 +0200
+++ mozilla-central/python/virtualenv/virtualenv.py	2017-05-22 10:10:39.112715085 +0200
@@ -1339,6 +1339,8 @@
         if py_executable_ext.lower() == '.exe':
             # python2.4 gives an extension of '.4' :P
             secondary_exe += py_executable_ext
+        if sys.platform in ('win32', 'cygwin') and py_executable_ext.lower() != '.exe':
+            secondary_exe += '.exe'
         if os.path.exists(secondary_exe):
             logger.warn('Not overwriting existing %s script %s (you must use %s)'
                         % (expected_exe, secondary_exe, py_executable))